### PR TITLE
[RF]AllTestsWasModifiedWithNewBaseTestsMethods<OlegCernis>

### PR DESCRIPTION
**Pull request by OlegCernis**
All metods in test was modified
[RF] - https://trello.com/c/bHCNxb7b

### CODE REVIEW

**Review by Snegafal — APPROVED** [review]

File: src/test/java/OlegCernisTest.java [hunk 1]
```diff
@@ -1,108 +1,59 @@
...
+    @Test
+    public void testVerifyChangingTempUnitInHeading_WhenSwitchTempUnitButton() {
+
+        openBaseURL();
+        click(fahrenheit);
+
+        String actualResult = getText(confirmTemp);
+
+        Assert.assertTrue(actualResult.contains("°F"));
+
```
Comment (larisabratukhina) [inline src/test/java/OlegCernisTest.java:46]: delete line

File: src/test/java/OlegCernisTest.java [hunk 2]
```diff
@@ -1,108 +1,59 @@
...
+    @Test
+    public void testVerifyThatLocationIsChangedInCurrentWeatherBlockAfterInputNewCity() {
+
+        final String newCity = "Chisinau, MD ";
+        final String expectedCity = "Chisinau, MD ";
+
+        openBaseURL();
+
+        String oldH2Header = getText(H2_CITY_COUNTRY_HEADER);
+        click(searchCityField);
```
Comment (larisabratukhina) [inline src/test/java/OlegCernisTest.java:27]: add line

Comment (OlegCernis) [reply src/test/java/OlegCernisTest.java:27]: first test was copied from IrinaZ class.

File: src/test/java/OlegCernisTest.java (hunk 1 above)
Comment (OlegCernis) [reply src/test/java/OlegCernisTest.java:46]: done

File: src/test/java/OlegCernisTest.java [hunk 3]
```diff
@@ -1,108 +1,59 @@
...
+        waitElementToBeVisible(SEARCH_DROPDOWN_MENU);
+        click(MOLDOVA_MD_CHOICE_IN_DROPDOWN_MENU);
+        waitTextToBeChanged(H2_CITY_COUNTRY_HEADER, oldH2Header);
+
+        Assert.assertEquals(expectedCity, newCity);
+    }
+
+    @Test
+    public void testVerifyChangingTempUnitInHeading_WhenSwitchTempUnitButton() {
+
```
[the remark this answers is not in the document]
Comment (OlegCernis) [reply src/test/java/OlegCernisTest.java:39]: done

File: src/test/java/OlegCernisTest.java [hunk 4]
```diff
@@ -1,108 +1,59 @@
...
+    final static By SEARCH_DROPDOWN_MENU = By.className("search-dropdown-menu");
+    final static By MOLDOVA_MD_CHOICE_IN_DROPDOWN_MENU =
+            By.xpath("//ul[@class = 'search-dropdown-menu']/li/span[text() = 'Chisinau, MD ']");
+    final static By fahrenheit = By.xpath("//div[text()= 'Imperial: °F, mph']");
+    final static By metric = By.xpath("//div[text()= 'Metric: °C, m/s']");
+    final static By confirmTemp = By.xpath("//div[@class = 'current-temp']/span");
+
+    @Test
+    public void testVerifyThatLocationIsChangedInCurrentWeatherBlockAfterInputNewCity() {
+
```
[the remark this answers is not in the document]
Comment (OlegCernis) [reply src/test/java/OlegCernisTest.java:20]: done

File: src/test/java/OlegCernisTest.java [hunk 5]
```diff
@@ -1,108 +1,59 @@
...
-//        openBaseURL();
-//        waitForGrayFrameDisappeared();
-//        click(metric, getWait5());
-//
-//        String actualResult2 = getText(confirmTemp,getDriver());
-//
-//        Assert.assertTrue(actualResult2.contains("°C"));
-//    }
-//}
+
```
[the remark this answers is not in the document]
Comment (OlegCernis) [reply src/test/java/OlegCernisTest.java:1]: there wasn't so mach lines.
image

File: src/test/java/OlegCernisTest.java [hunk 6]
```diff
@@ -1,108 +1,59 @@
...
+import org.openqa.selenium.By;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import base.BaseTest;
+
+public class OlegCernisTest extends BaseTest {
+    final static By searchCityField = By.xpath("//div[@id='weather-widget']//input[@placeholder= 'Search city']");
```
[the remark this answers is not in the document]
Comment (OlegCernis) [reply src/test/java/OlegCernisTest.java:8]: done

File: src/test/java/OlegCernisTest.java (hunk 2 above)
Comment (OlegCernis) [reply src/test/java/OlegCernisTest.java:28]: Сделал, спасибо за подсказку

**Review by larisabratukhina — APPROVED** [review]

**Pull request merged**